starknet_devnet/starknet_wrapper.py

```diff
@@ -988,3 +987,48 @@ async def is_deployed(self, address: int) -> bool:
         cached_state = self.get_state().state
         class_hash = await cached_state.get_class_hash_at(address)
         return bool(class_hash)
+
+    async def abort_blocks(self, starting_block: StarknetBlock) -> str:
+        """
+        Abort blocks.
+        """
+        # Check if blocks can be aborted in fork mode
+        if (
+            self.config.fork_block
+            and self.config.fork_block >= starting_block.block_number
+            or starting_block.block_number == 0
```

Comment:
`starting_block.block_number == 0` is to prevent the abortion of genesis block

Reply:
I meant, "genesis block" instead of "origin block" (I edited the message). So my point is that genesis block isn't necessarily at 0.

Reply:
Fixed in 74b90cba85ac60e019b19ec787a2586f1052070b

Reply:
In the commit that you linked, I see it only sets genesis_block_number if not forking (actually, if it's any falsy object; e.g. None and 0). Shouldn't it always be set?

Also, instead of adding more properties to starknet_wrapper, can this be added to the `DevnetBlocks` class?

Reply:
From now on, let's refer to the first block in Devnet as `devnet_genesis_block`. Let's not allow aborting it (whether it is really a genesis (first block) or just the first block after forking).

Reply:
Fixed in 8831b87f1a2ac2c867d30087f794974d6366114c